examples/spack/Dockerfile

```diff
@@ -14,22 +13,26 @@ FROM debian9
 # [1]: https://spack.readthedocs.io/en/latest/getting_started.html
 # [2]: https://spack.readthedocs.io/en/latest/workflows.html#using-spack-to-create-docker-images
 
-# Packages needed to install Spack [1].
-#
```

Comment:
Why remove this comment?

Reply:
This appears to be a mistake. Good catch.